grommet/grommet
# Updated Build script

Updated build script to exclude test files and portal.js from distribution folder



#### What does this PR do?
 Removes test files and portal.js from dist folder

#### Where should the reviewer start?
`package.json` line 32

#### What testing has been done on this PR?
I have manually tested it by running `yarn build` and checking dist folder to confirm that the right files were excluded
#### How should this be manually tested?
Run `yarn build` and then browse through the `grommet/dist` directory to confirm that there are no test files included

#### What are the relevant issues?
#4118 and #4119 

### Relevant Screenshots
`grommet/dist` directory before updating the script 
<img width="779" alt="Screen Shot 2020-05-28 at 3 54 19 PM" src="https://user-images.githubusercontent.com/49867133/83187254-e0faf500-a0fb-11ea-8ab0-d19cd18b5026.png">
`grommet/dist` directory  after updating script
<img width="789" alt="Screen Shot 2020-06-01 at 4 19 34 PM" src="https://user-images.githubusercontent.com/49867133/83450878-18cbaa80-a424-11ea-92b2-1d98d8668593.png">


`grommet/dist/utils` directory before updating script
<img width="784" alt="Screen Shot 2020-05-28 at 3 54 34 PM" src="https://user-images.githubusercontent.com/49867133/83187298-f3752e80-a0fb-11ea-9480-fc00c83cd3b4.png">
`grommet/dist/utils` directory after updating script
<img width="790" alt="Screen Shot 2020-06-01 at 4 19 57 PM" src="https://user-images.githubusercontent.com/49867133/83450868-12d5c980-a424-11ea-8bf0-820bc83035fb.png">





#### Do the grommet docs need to be updated?
No
#### Should this PR be mentioned in the release notes?
No
#### Is this change backwards compatible or is it a breaking change?
It is backwards compatible
